rust/decentraland-godot-lib/src/realm/scene_entity_coordinator.rs

```diff
@@ -131,7 +28,7 @@ struct SceneEntityCoordinator {
 
     global_desired_entities: Vec<EntityBase>,
     requested_entity: HashMap<u32, EntityBase>,
-    cache_scene_data: HashMap<String, EntityDefinitionJson>, // entity_id to SceneData
+    cache_scene_data: HashMap<String, Arc<SceneEntityDefinition>>, // entity_id to SceneData
```

Comment:
In case it needs to be mutable, a Mutex would be required (or RwLock), it depends on the situation.
The Arc is implemented to be available between threads, without copying all the contents (content mapping is already an Arc also). Without Mutex/RwLock, it means the content that holds is immutable.

Reply:
The threads involved are the scene thread, main thread, and content loading thread.